frontend/src/static/css/style.scss

```diff
@@ -252,19 +252,35 @@ header{
         }
 
         .untouched{
-            $color:mui-color("black");
-            border-style:solid;
-            border-color:$color;
-            border-width:1px 0;
             margin-top:1rem;
 
             .closer{
                 cursor:pointer;
                 padding:0.5rem 1rem;
 
-                span{
-                    color:$color;
-                    font-weight:bold;
+                svg{
+                    background-color: #bfbdbd;
+                    color: white;
+                    width: 2.5em;
+                    border-radius: 5px;
+                }
```

Comment:
dun use `svg` because this is how fontawesome renders the font, it might be that they change their rendering format, then this will screw up.

use `.fa-ellipsis-h` instead